apps/test/integration/levelTests.js

```diff
@@ -25,6 +25,7 @@ import arrowDisplay from '@cdo/apps/templates/arrowDisplayRedux';
 import FirebaseStorage from '@cdo/apps/storage/firebaseStorage';
 import LegacyDialog from '@cdo/apps/code-studio/LegacyDialog';
 import loadSource from './util/loadSource';
+import projectRedux from '@cdo/apps/code-studio/projectRedux';
```

Comment:
super nit / more of a general javascript tip: since we're importing the default export here, we can name the import whatever we want. so here we could have

```javascript
import project from '@cdo/apps/code-studio/projectRedux';

// no longer have to rename projectRedux --> project here
registerReducers({project});
```

not super necessary here, but a useful tip to keep in mind!

Reply:
Thanks! I originally had import renamed as project, but on line 20, project name is already used.
Thanks so much for your help and review!